README.md

```diff
@@ -209,6 +209,8 @@ options covering different styles. File an issue, or even better, submit a PR.
 * When you run rustfmt, place a file named `rustfmt.toml` or `.rustfmt.toml` in
   target file directory or its parents to override the default settings of
   rustfmt.
+* You can dump the default configuration with
```

Comment:
It would be good to expand on this a little bit - something like "Rustfmt is configured using a toml file. You can..."

Reply:
Do you think it is necessary? It is already mentioned in the first paragraph of the "Configuring Rustfmt" section and implied by the second bullet under "Tips".

(By the way, thanks for taking the time to review my silly PRs! :)

Reply:
I think the tips section should be readable by itself (I often refer people to the tips section directly with a URL). Maybe you don't need to be as verbose as my suggestion given the first tip, but you should maybe mention "rustfmt.toml' or something (currently the first tip talks about rustfmt.toml and settings, and the second tip talks about configuration, it is not clear they are the same thing).